templates/rsyslog.conf.erb

```diff
@@ -67,6 +67,13 @@ $Umask <%= scope.lookupvar('rsyslog::umask') %>
 $OmitLocalLogging on
 <% end -%>
 
+# Settings for imjournal
+<% if scope.lookupvar('rsyslog::im_journal_statfile') -%>
+$IMJournalStateFile <%=scope.lookupvar('rsyslog::im_journal_statfile') %>
```

Comment:
This doesn't appear to be a valid parameter - perhaps it needs to be added to `manifests/init.pp` and `manifests/params.pp` ?